sources/tests/.env-e2e-tests

```diff
@@ -8,4 +8,7 @@ SWAGGER_ENABLED=false
 POSTGRES_DB=aest_e2e_tests
 POSTGRES_USER=admin_e2e_tests
 POSTGRES_PASSWORD=aest_sim_e2e_tests
-POSTGRES_PORT=5430
\ No newline at end of file
+POSTGRES_PORT=5430
+
+#Zeebe
+ZEEBE_ADDRESS=zeebe:26500
```

Comment:
Is this needed since it is already hard coded at sources/tests/workflow-docker-compose.yml